src/lib.rs

```diff
@@ -11,6 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+#![warn(clippy::pedantic)]
```

Comment:
💭 Does adding this here mean it is inherited by all the other source files in the crate or no?

Reply:
Yes, it turns out that it is inherited by all other source files - I will remove it from the others